src/API/Data/AuthorizationRepository.cs

```diff
@@ -132,25 +126,12 @@ private static async Task<Result<(Person requestor, Person target),Error>> Fetch
                 ? Pipeline.Success(PermsGroups.All)
                 : Pipeline.Success(EntityPermissions.Get);
                 
-        public static Result<EntityPermissions,Error> ResolveUnitPermissions(Person requestor, int unitId)
+        public static Result<EntityPermissions,Error> ResolveUnitAndUnitMemberPermissions(Person requestor, int unitId)
```

Comment:
This name is a little long. Maybe `DetermineUnitManagementPermissions`?

Reply:
this 796dbcbe275fab26330aad5761e9facd2e595e7e would change the method name.